### PR TITLE
chore(docs): Update contributing.md

### PR DESCRIPTION
**Pull request by mven**
First iteration to make it better.

## Overview
- add steps to spin up the server
- change/improve some wording
- add more structure to the process

### CODE REVIEW

**Review by tw15egan — APPROVED** [review]
🙌 Thank you!

**Review by aledavila — APPROVED** [review]

File: docs/contributing.md [hunk 1]
```diff
@@ -78,73 +80,61 @@ button
...
-```
-
-*Do not submit pull requests from the `master` branch of your fork.*
+### 5. Start the server
 
 ```
-git checkout -b { YOUR_BRANCH_NAME }
-git add .
-git commit -m "fix(table): IE11 positioning error" -m "Fixes #34"
+run run start
```
Comment (asudoh) [inline docs/contributing.md]: If this section means development server here, we use either of the following: `yarn dev` or `gulp serve`.

File: docs/contributing.md [hunk 2]
```diff
@@ -78,73 +80,61 @@ button
...
 ```
 
-* [Close a commit via commit message](https://help.github.com/articles/closing-issues-via-commit-messages/)
+Once it's done building, you can start editing source code or creating new components. The system is set up to automatically bundle your changes/additions. Visit http://localhost:8080 to see the changes happen on the fly.
```
Comment (asudoh) [inline docs/contributing.md]: We use `localhost:3000` for running the dev server.

**Review by asudoh — COMMENTED** [review]
Thanks @mven for making an improvement here!! Just a couple of comments.

File: docs/contributing.md (hunk 1 above)
Comment (mven) [reply docs/contributing.md]: thanks -- but `npm run serve` over `gulp serve` should do the same?

File: docs/contributing.md (hunk 2 above)
Comment (mven) [reply docs/contributing.md]: ✅

File: docs/contributing.md (hunk 1 above)
Comment (asudoh) [reply docs/contributing.md]: Good question - We didn't do for historical reason (running BrowserSync vs. raw server), but given we have drifted away from BrowserSync, I think it'll make sense for us to update `package.json` for that.

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks alot @mven for making the quick update!

**Pull request merged**